cmd/buildah/config.go

```diff
@@ -17,99 +17,71 @@ const (
 )
 
 var (
-	configurationFlags = []cli.Flag{
+	configFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "name",
+			Usage: "name or ID of the working container",
+		},
+		cli.StringFlag{
+			Name:  "root",
+			Usage: "root directory of the working container",
+		},
+		cli.StringFlag{
+			Name:  "link",
+			Usage: "symlink to the root directory of the working container",
+		},
 		cli.StringFlag{
 			Name:  "author",
-			Usage: "image author contact information",
+			Usage: "image author contact `information`",
 		},
 		cli.StringFlag{
 			Name:  "created-by",
-			Usage: "description of how the image was created",
+			Usage: "`description` of how the image was created",
 			Value: DefaultCreatedBy,
 		},
 		cli.StringFlag{
 			Name:  "arch",
-			Usage: "image target architecture",
+			Usage: "image target `architecture`",
 		},
 		cli.StringFlag{
 			Name:  "os",
-			Usage: "image target operating system",
+			Usage: "image target operating `system`",
```

Comment:
The --help output for that looks like
`   --os operating system     image target operating system`
which looked a bit weird to me.